### DISCUSSION TITLE
A built-in keyboard like this, Bitwarden can add?

### DISCUSSION THREAD

**Discussion by biange200** [community/community#46912] [Mobile] [answered]
![Screenshot_20230210_084028.jpg](https://user-images.githubusercontent.com/37582806/217971945-94c39e60-b8da-4af7-b3ab-abf1bf91f044.jpg)

![Screenshot_20230210_083305.jpg](https://user-images.githubusercontent.com/37582806/217972236-fec6c06b-2540-4ca3-9ba8-1ed86e81e2aa.jpg)

I have apps that don't support auto-fill, Bitwarden can add this feature?

**Comment by stevepopovich** [comment, marked as answer]
Hi @biange200 Seems like this is feedback for BitWarden, not the GitHub Mobile app. You could try opening an issue on that repo directly: https://github.com/bitwarden/mobile

**Comment by biange200** [comment]
Sorry, I sent it wrong



---- Replied Message ----
| From | Steven ***@***.***> |
| Date | 02/13/2023 22:11 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [community/community] A built-in keyboard like this, Bitwarden can add? (Discussion #46912) |

Hi @biange200 Seems like this is feedback for BitWarden, not the GitHub Mobile app. You could try opening an issue on that repo directly: https://github.com/bitwarden/mobile
